src/exo/range_analysis.py

```diff
@@ -0,0 +1,82 @@
+from .LoopIR import LoopIR
+
+
+class AffineIndexRangeAnalysis:
```

Comment:
This class needs a comment here in explaining what it does.

Reply:
Done